ingestion/src/metadata/ingestion/source/dashboard/tableau/queries.py

```diff
@@ -23,9 +23,15 @@
     nodes {{
       id
       name
+      hasExtracts
       upstreamDatasources{{
         id
         name
+        description
+        hasExtracts
```

Comment:
is `hasExtracts` being used anywhere? If not we can remove the same

Reply:
As I wrote in the original PR my idea was to add it, and then eventually in a future iteration consider to see how to see if a data-source has extracts or not, but as I believe that this is another topic I will simply remove it for now:)